### PR TITLE
Correction and updates to CDC known limitations

### PR DESCRIPTION
**Pull request by kathancox**
Fixes DOC-9230

Updates and corrections to the CDC known limitations. A couple of these were out of date.

### CODE REVIEW

**Review by miretskiy — APPROVED** [review]

File: src/current/v23.2/create-changefeed.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ Option | Value | Description
...
+<a name="on-error"></a>`on_error` | `pause` / `fail` |  Use `on_error=pause` to pause the changefeed when encountering **non**-retryable errors. `on_error=pause` will pause the changefeed instead of sending it into a terminal failure state. **Note:** Retryable errors will continue to be retried with this option specified. <br><br>Use with [`protect_data_from_gc_on_pause`](#protect-pause) to protect changes from [garbage collection]({% link {{ page.version.version }}/configure-replication-zones.md %}#gc-ttlseconds).<br><br>If a changefeed with `on_error='pause` is running when a watched table is [truncated]({% link {{ page.version.version }}/truncate.md %}), the changefeed will pause but will not be able to resume reads from that table. Using [`ALTER CHANGEFEED`]({% link {{ page.version.version }}/alter-changefeed.md %}) to drop the table from the changefeed and then [resuming the job]({% link {{ page.version.version }}/resume-job.md %}) will work, but you cannot add the same table to the changefeed again. Instead, you will need to [create a new changefeed](#start-a-new-changefeed-where-another-ended) for that table.<br><br>Default: `on_error=fail`
```
Comment (florence-crl) [inline src/current/v23.2/create-changefeed.md:179]: Should the single-quote be removed or should another single-quote be added after `pause` in:
> `on_error='pause`
?

File: src/current/v22.2/create-changefeed.md [hunk 2]
```diff
@@ -163,7 +163,7 @@ Option | Value | Description
...
+<a name="on-error"></a>`on_error` | `pause` / `fail` |  Use `on_error=pause` to pause the changefeed when encountering **non**-retryable errors. `on_error=pause` will pause the changefeed instead of sending it into a terminal failure state. **Note:** Retryable errors will continue to be retried with this option specified. <br><br>Use with [`protect_data_from_gc_on_pause`](#protect-pause) to protect changes from [garbage collection](configure-replication-zones.html#gc-ttlseconds).<br><br>If a changefeed with `on_error='pause'` is running when a watched table is [truncated](truncate.html), the changefeed will pause but will not be able to resume reads from that table. Using [`ALTER CHANGEFEED`](alter-changefeed.html) to drop the table from the changefeed and then [resuming the job](resume-job.html) will work, but you cannot add the same table to the changefeed again. Instead, you will need to [create a new changefeed](#start-a-new-changefeed-where-another-ended) for that table.<br><br>Default: `on_error=fail`
```
[the remark this answers is not in the document]
Comment (florence-crl) [reply src/current/v22.2/create-changefeed.md:166]: Should the single-quotes be included: `on_error='pause'` ?

File: src/current/_includes/v23.2/known-limitations/cdc.md [hunk 3]
```diff
@@ -1,11 +1,8 @@
...
 - Webhook sinks only support HTTPS. Use the [`insecure_tls_skip_verify`]({% link {{ page.version.version }}/create-changefeed.md %}#tls-skip-verify) parameter when testing to disable certificate verification; however, this still requires HTTPS and certificates. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73431)
-- [Webhook sinks]({% link {{ page.version.version }}/changefeed-sinks.md %}#webhook-sink) only support emitting `JSON`. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73432)
-- There is no concurrency configurability for [webhook sinks]({% link {{ page.version.version }}/changefeed-sinks.md %}#webhook-sink). [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73430)
+- Formats for changefeed messages are not supported by all changefeed sinks. Refer to the [Changefeed Sinks]({% link {{ page.version.version }}/changefeed-sinks.md %}) page for detail on compatible formats with each sink and the [`format`]({% link {{ page.version.version }}/create-changefeed.md %}) option to specify a changefeed message format. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73432)
```
Comment (florence-crl) [inline src/current/_includes/v23.2/known-limitations/cdc.md:4]: Please change "for detail on compatible formats" to "for details on compatible formats".

File: src/current/_includes/v23.1/known-limitations/cdc.md [hunk 4]
```diff
@@ -1,11 +1,8 @@
...
 - Webhook sinks only support HTTPS. Use the [`insecure_tls_skip_verify`]({% link {{ page.version.version }}/create-changefeed.md %}#tls-skip-verify) parameter when testing to disable certificate verification; however, this still requires HTTPS and certificates. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73431)
-- [Webhook sinks]({% link {{ page.version.version }}/changefeed-sinks.md %}#webhook-sink) only support emitting `JSON`. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73432)
-- There is no concurrency configurability for [webhook sinks]({% link {{ page.version.version }}/changefeed-sinks.md %}#webhook-sink). [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73430)
+- Formats for changefeed messages are not supported by all changefeed sinks. Refer to the [Changefeed Sinks]({% link {{ page.version.version }}/changefeed-sinks.md %}) page for detail on compatible formats with each sink and the [`format`]({% link {{ page.version.version }}/create-changefeed.md %}) option to specify a changefeed message format. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73432)
```
Comment (florence-crl) [inline src/current/_includes/v23.1/known-limitations/cdc.md:4]: Please change "for detail on compatible formats" to "for details on compatible formats".

File: src/current/_includes/v22.2/known-limitations/cdc.md [hunk 5]
```diff
@@ -1,10 +1,7 @@
...
 - Webhook sinks only support HTTPS. Use the [`insecure_tls_skip_verify`](create-changefeed.html#tls-skip-verify) parameter when testing to disable certificate verification; however, this still requires HTTPS and certificates. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73431)
-- [Webhook sinks](changefeed-sinks.html#webhook-sink) and [Google Cloud Pub/Sub sinks](changefeed-sinks.html#google-cloud-pub-sub) only have support for emitting `JSON`. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73432)
-- There is no concurrency configurability for [webhook sinks](changefeed-sinks.html#webhook-sink). [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73430)
+- Formats for changefeed messages are not supported by all changefeed sinks. Refer to the [Changefeed Sinks]({% link {{ page.version.version }}/changefeed-sinks.md %}) page for detail on compatible formats with each sink and the [`format`]({% link {{ page.version.version }}/create-changefeed.md %}) option to specify a changefeed message format. [Tracking GitHub Issue](https://github.com/cockroachdb/cockroach/issues/73432)
```
Comment (florence-crl) [inline src/current/_includes/v22.2/known-limitations/cdc.md:4]: Please change "for detail on compatible formats" to "for details on compatible formats".

**Review by florence-crl — APPROVED** [review]
LGTM 2 minor corrections:
- should `pause` be single-quoted?
- make "detail" plural

**Comment by kathancox** [comment]
Feedback added! TFTRs

File: src/current/v23.2/create-changefeed.md (hunk 1 above)
Comment (kathancox) [reply src/current/v23.2/create-changefeed.md]: You can successfully run that option with the value in single quotes or not.

**Pull request merged**